.github/workflows/ci_e2e.yaml

```diff
@@ -0,0 +1,203 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '53 0 * * SAT' # Every Saturday at 00:53 UTC
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install tox
+        # TODO: Consider replacing with custom image on self-hosted runner OR pinning version
+        run: python3 -m pip install tox
+      - name: Run linters
+        run: tox run -e lint
+
+  build:
+    name: Build charms
+    uses: canonical/data-platform-workflows/.github/workflows/build_charms_with_cache.yaml@v2
+
+  integration-test:
```

Comment:
**thought** maybe we could skip this, since we are anyway already running these in "normal" CI/CD pipelines.

Reply:
I removed these tests.